lib/_http_outgoing.js

```diff
@@ -657,12 +657,11 @@ OutgoingMessage.prototype._flushOutput = function _flushOutput(socket) {
   var output = this.output;
   var outputEncodings = this.outputEncodings;
   var outputCallbacks = this.outputCallbacks;
-  socket.cork();
+
   for (var i = 0; i < outputLength; i++) {
     ret = socket.write(output[i], outputEncodings[i],
                        outputCallbacks[i]);
   }
-  socket.uncork();
```

Comment:
However, https://github.com/brendanashworth/io.js/blob/7f4698961ec7e035974abd277f9607ead6e58a99/lib/_http_outgoing.js#L633-L648 calls `_flushOutput`. For simmetry, we should `cork()` there as well.